### ISSUE TITLE
Source code import: newlines should be preserved in assignments, maybe added automatically

### ISSUE THREAD

**Issue by GitMensch** [fesch/Structorizer.Desktop#446]
Ideally we include the newlines within a Structorizer element as Structorizer continuation.

Given the sample:

```cobol
 01 hex-tab pic x(512) value
 "000102030405060708090a0b0c0d0e0f" &
 "101112131415161718191a1b1c1d1e1f" &
 "202122232425262728292a2b2c2d2e2f" &
 "303132333435363738393a3b3c3d3e3f" &
 "404142434445464748494a4b4c4d4e4f" &
 "505152535455565758595a5b5c5d5e5f" &
 "606162636465666768696a6b6c6d6e6f" &
 "707172737475767778797a7b7c7d7e7f" &
 "808182838485868788898a8b8c8d8e8f" &
 "909192939495969798999a9b9c9d9e9f" &
 "a0a1a2a3a4a5a6a7a8a9aaabacadaeaf" &
 "b0b1b2b3b4b5b6b7b8b9babbbcbdbebf" &
 "c0c1c2c3c4c5c6c7c8c9cacbcccdcecf" &
 "d0d1d2d3d4d5d6d7d8d9dadbdcdddedf" &
 "e0e1e2e3e4e5e6e7e8e9eaebecedeeef" &
 "f0f1f2f3f4f5f6f7f8f9fafbfcfdfeff".
```

The current translation is

```
var hex_tab: String <- "000102030405060708090a0b0c0d0e0f" +"101112131415161718191a1b1c1d1e1f" +"202122232425262728292a2b2c2d2e2f" +"303132333435363738393a3b3c3d3e3f" +"404142434445464748494a4b4c4d4e4f" +"505152535455565758595a5b5c5d5e5f" +"606162636465666768696a6b6c6d6e6f" +"707172737475767778797a7b7c7d7e7f" +"808182838485868788898a8b8c8d8e8f" +"909192939495969798999a9b9c9d9e9f" +"a0a1a2a3a4a5a6a7a8a9aaabacadaeaf" +"b0b1b2b3b4b5b6b7b8b9babbbcbdbebf" +"c0c1c2c3c4c5c6c7c8c9cacbcccdcecf" +"d0d1d2d3d4d5d6d7d8d9dadbdcdddedf" +"e0e1e2e3e4e5e6e7e8e9eaebecedeeef" +"f0f1f2f3f4f5f6f7f8f9fafbfcfdfeff"
```

and the better version would be something like

```
var hex_tab: String <-  \
 "000102030405060708090a0b0c0d0e0f" \
 + "101112131415161718191a1b1c1d1e1f" \
 + "202122232425262728292a2b2c2d2e2f" \
 + "303132333435363738393a3b3c3d3e3f" \
 + "404142434445464748494a4b4c4d4e4f" \
 + "505152535455565758595a5b5c5d5e5f" \
 + "606162636465666768696a6b6c6d6e6f" \
 + "707172737475767778797a7b7c7d7e7f" \
 + "808182838485868788898a8b8c8d8e8f" \
 + "909192939495969798999a9b9c9d9e9f" \
 + "a0a1a2a3a4a5a6a7a8a9aaabacadaeaf" \
 + "b0b1b2b3b4b5b6b7b8b9babbbcbdbebf" \
 + "c0c1c2c3c4c5c6c7c8c9cacbcccdcecf" \
 + "d0d1d2d3d4d5d6d7d8d9dadbdcdddedf" \
 + "e0e1e2e3e4e5e6e7e8e9eaebecedeeef" \
 + "f0f1f2f3f4f5f6f7f8f9fafbfcfdfeff"
```

The main reason is the scaling of the Structorizer Content:

![grafik](https://user-images.githubusercontent.com/6699539/32219143-7e8fe852-be2d-11e7-8d98-af0944451387.png)

vs.

![grafik](https://user-images.githubusercontent.com/6699539/32219114-65aab088-be2d-11e7-80a0-149940c71701.png)

We may even add newlines automatically during import if the line length of a Structorizer element is > a given number (import option) and we parse a spacing character.

**Comment by codemanyak** [comment]
We may need a better expression parsing for this.
I was going to recommend the Find & Relace tool but it fails to insert newlines. Moreover, it got stuck in an endless loop 😞 .
So okay, I'll try to find a heuristic solution.

**Comment by codemanyak** [comment]
I think, this issue was resolved with the fixes for #416, #419, and #626, delivered with version 3.28-11.
